CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ All notable changes to this project will be documented in this file.
 - Upgrade to `iexec-result-proxy-library` 8.5.0. (#603)
 - Upgrade to `iexec-sms-library` 8.6.0. (#603)
 - Upgrade to `iexec-core-library` 8.5.0. (#603)
+- Upgrade to `eclipse-temurin:11.0.24_8-jre-focal`. (#610)
```

Comment:
This should be in the [NEXT] section, not here
8.5.0 has been released and won't see its content updated :-)
Updated content will lead to the production of the next release (8.5.x or 8.6.0)